src/test/java/com/cdancy/bitbucket/rest/features/HookApiMockTest.java

```diff
@@ -25,11 +25,11 @@
 import com.cdancy.bitbucket.rest.TestUtilities;
 import com.cdancy.bitbucket.rest.domain.repository.HookSettings;
 import com.google.common.collect.ImmutableMap;
-import com.google.gson.internal.LinkedTreeMap;
 import com.squareup.okhttp.mockwebserver.MockResponse;
 import com.squareup.okhttp.mockwebserver.MockWebServer;
 import org.testng.annotations.Test;
 
+import java.util.HashMap;
```

Comment:
:warning: Extra separation in import group before 'java.util.HashMap'
Wrong lexicographical order for 'java.util.HashMap' import. Should be before 'org.testng.annotations.Test'. (#QR177:32#)-CHECKSTYLE_CustomImportOrderCheck